ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/fabric-topology.md

```diff
@@ -422,8 +422,40 @@ l2leaf:
           id: < integer >
           mgmt_ip: < IPv4_address/Mask >
           l3leaf_interfaces: [ < ethernet_interface_8 >, < ethernet_interface_8 > ]
+
+    # node_group_3, will result in Active/Active connection to L3LEAFs.
+    < node_group_3 >:
+      parent_l3leafs: [ DC1-SVC3A ]
+      mlag: false
```

Comment:
This `mlag: false` is not needed. Actually we might run MLAG on the l2leaf and A/A on the l3leaf, so this is misleading.

Reply:
Not required anymore with commit e3c7310